modules/data.atmosphere/R/met.process.R

```diff
@@ -270,8 +282,34 @@ met.process <- function(site, input_met, start_date, end_date, model,
     reg.model.xml <- system.file(paste0("register.", model, ".xml"), package = paste0("PEcAn.",model))
     reg.model <- XML::xmlToList(XML::xmlParse(reg.model.xml))
     
-    met2model.result = list()
-    for (i in 1:length(ready.id[[1]])) {
+      met2model.result = list()
+    # Here I assume ensembles have one input id and multiple files related to them
+    if (!is.null(register$ensemble)){
```

Comment:
So if met.process was previously working for other ensemble met (e.g. GEFS, PalEON), why do we need a new/different behavior here? Wasn't the existing looping over ID's done specifically to support ensemble met?